site/pages/Index.vue

```diff
@@ -1,181 +1,171 @@
 <template>
   <div>
     <section
-      class="bg-hero-beams bg-no-repeat bg-cover lg:bg-contain flex justify-center px-8 lg:px-0 lg:pt-12 2xl:pt-0 pb-16"
+      class="relative flex h-[1064px] justify-center bg-hero-beams bg-cover bg-center bg-no-repeat py-32 text-center sm:h-[1024px] lg:bg-contain 2xl:h-[1124px] 2xl:py-24"
     >
-      <div class="flex flex-col items-center w-[1400px]">
+      <div class="container flex flex-col items-center gap-9 text-center">
+        <h1
+          class="px-3 text-center text-6xl leading-5xl text-sea-green sm:px-0 lg:text-7xl lg:leading-6xl"
+        >
+          <span class="font-bold">
+            Explore Rollups
+            <br />
+            Effortlessly
+          </span>
+        </h1>
+        <h2 class="px-[10%] text-xl lg:text-2xl xl:px-[20%]">
+          Beam your ERC20 assets between EVM compatible rollups with certainty, speed and security
+        </h2>
+        <a
+          class="rounded-[67px] bg-neon-yellow py-4 px-8 text-2xl text-black transition-shadow hover:shadow-2xl"
+          href="https://testnet.beamerbridge.com"
+          target="_blank"
+        >
+          <span class="inline whitespace-nowrap">Start Beaming</span>
+        </a>
+      </div>
+
+      <!-- transitioned element -->
+      <div
```

Comment:
:+1:

Reply:
I reduced the paddings/margins as much as possible on mobile.
I think on mobile there is no way that the "Simple and Secure" element can be visible since the first section is taking up the whole screen for most of the mobile devices.
Check it out now and tell me if its improved.

Reply:
Okay no worries. Seems fine like this to me.